x-pack/plugins/security_solution/public/resolver/view/process_event_dot.tsx

```diff
@@ -327,8 +329,18 @@ const UnstyledProcessEventDot = React.memo(
     const grandTotal: number | null = useSelector((state: ResolverState) =>
       selectors.statsTotalForNode(state)(node)
     );
-
     const nodeName = nodeModel.nodeName(node);
+    const processEvent = useSelector((state: ResolverState) =>
+      nodeDataModel.firstEvent(selectors.nodeDataForID(state)(String(node.id)))
```

Comment:
Isn't the node id already a string? When isn't it? Or can it be numerical, can't remember...

Reply:
According to the type, it can be a number. It actually should be just a string I believe, but we can revisit at a later date as part of https://github.com/elastic/kibana/issues/128315